torch/hub.py

```diff
@@ -53,6 +56,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
                 sys.stderr.write('\n')
 
+__all__ = [
```

Comment:
in line with the [documented API for `torch.hub`](https://pytorch.org/docs/master/hub.html#torch.hub.download_url_to_file) so LGTM